lib/msal-common/src/request/BaseAuthRequest.ts

```diff
@@ -48,4 +49,5 @@ export type BaseAuthRequest = {
     tokenQueryParameters?: StringDict;
     storeInCache?: StoreInCache;
     scenarioId?: string;
+    reqCnf?: ReqCnfData;
```

Comment:
I don't think this needs to be of type object, this can be string

Reply:
+1

Reply:
Got it.. thanks!